### PR TITLE
New

### PR DESCRIPTION
**Pull request by neha07kumari**

### CODE REVIEW

**Comment by neha07kumari** [comment]
I have added Day 6 hackerrank Question's solution in C from 30daysofcode . Please check it. @Vishruth-S

File: Hackerrank_problems/Extra Long Factorials/solution.C [hunk 1]
```diff
@@ -0,0 +1,26 @@
+int main()
```
Comment (Vishruth-S) [inline Hackerrank_problems/Extra Long Factorials/solution.C:1]: As mentioned in #312, Please add comments explaining the logic to this solution of `Extra long factorials`
The other solutions are fine.

**Review by Vishruth-S — CHANGES_REQUESTED** [review]

**Comment by neha07kumari** [comment]
Yes I'm adding it .

File: Hackerrank_problems/Extra Long Factorials/solution.C (hunk 1 above)
Comment (neha07kumari) [reply Hackerrank_problems/Extra Long Factorials/solution.C:1]: added it.

**Review by Vishruth-S — APPROVED** [review]
Good job 👍  Thanks

**Pull request merged**